doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10095 (nRF5340 PDK)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|: ??? please check
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.50b
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v10.6.0
+     - `nRF Command Line Tools`_
+   * - nRF Connect for Desktop
+     - v3.3.0 or later
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
+
+As IDE, we recommend to use |SES| (Nordic Edition) version 4.44a.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`cloud_client` - shows how to connect to and communicate with a cloud service using the generic :ref:`cloud_api_readme`.
+  * :ref:`https_client` - shows how to provision a TLS certificate and connect to an HTTPS server.
+  * :ref:`serial_lte_modem` - demonstrates sending AT commands between a host and a client device.
+    The sample is an enhancement to the :ref:`at_client_sample` sample.
+
+* Added the following libraries:
+
+  * :ref:`lib_aws_iot` - enables applications to connect to and exchange messages with the AWS IoT message broker.
+    The library supports TLS-secured MQTT transmissions and firmware over-the-air upgrades.
+  * :ref:`modem_key_mgmt` - provides functions to provision security credentials to the nRF9160 modem.
+    The library replaces the ``nrf_inbuilt_key`` APIs from the :ref:`nrfxlib:bsdlib`.
+  * :ref:`lib_zzhc` - implements the self-registration functionality that is required to connect to the China Telecom network.
+  * :ref:`supl_client` - integrates the externally hosted SUPL client library.
+    This library implements A-GPS data downloading from a SUPL server.
+
+
+Updated samples and applications
+--------------------------------
+
+* :ref:`asset_tracker`:
+
+  * Added high/low threshold configuration command handling for sensor data. ??? what? Added handling for commands that configure the threshold? Added command handling for threshold configurations?
```

Comment:
Thresholds can be set so that only sensor data below or above certain thresholds is sent.
@jayteemo Is the support for this in the cloud UI in place? Could you add examples of commands to set thresholds, intervals and send AT commands somewhere publicly available?

Reply:
@ru-fu that is correct.
Example JSON command:
```
{
  "appId": "TEMP",
  "messageType": "CFG_SET",
  "data": {
    "enable": true,
    "thresh_lo": 20,
    "thresh_hi": 50
  }
} 
```
The above JSON can be sent to the device through the device's `Terminal` card on nrfcloud.com.
The device would then only report temperatures less than 20 or greater than 50.

Reply:
example AT command sent from the `Terminal` card on nrfcloud.com: 
`{"appId":"MODEM", "messageType":"CMD", "data":"AT+CFUN?\r\n"}`
AT response is then displayed in the `Terminal` card.

What is the desired _public place_ for such examples?